mlflow/store/cli.py

```diff
@@ -88,29 +92,31 @@ def _file_infos_to_json(file_infos):
 
 
 @commands.command("download")
-@click.option("--run-id", "-r", required=True,
+@click.option("--run-id", "-r",
               help="Run ID from which to download")
 @click.option("--artifact-path", "-a",
               help="If specified, a path relative to the run's root directory to download")
-def download_artifacts(run_id, artifact_path):
+@click.option("--artifact-uri", "-u",
+              help="URI pointing to the artifact file or artifacts directory.")
```

Comment:
Maybe we can say that this is an alternative to passing run-id and artifact-path